### DISCUSSION TITLE
How are players meant to retry the "Drain Sewers to find Statue" mission to unlock the Turbo JET-Board in Haven City secret?

### DISCUSSION THREAD

**Discussion by awkspace** [open-goal/jak-project#3431] [Q&A] [answered]
I didn't notice that this was a challenge in the secret menu until long after I had done the mission.

**Comment by dallmeyer** [comment, marked as answer]
level select (it's unlocked as a secret in the original game)

**Reply by awkspace** [reply]
I thought that level select only allowed you to start a new save, not modify an existing one.

**Reply by ManDude** [reply]
You don't need to modify an existing save, the secrets are unlocked for every savefile.
